models/gameModels.js

```diff
@@ -48,14 +49,37 @@ exports.fetchReviewById = (id) => {
 
 exports.fetchCommentsFromReview = (id) => {
   let queryString = "SELECT * FROM comments";
-  const query =[];
+  const query = [];
 
   if (id !== undefined) {
     queryString += ` WHERE review_id = $1 ORDER BY created_at DESC`;
-    query.push(id)
+    query.push(id);
   }
 
   return db.query(queryString, query).then((results) => {
     return results.rows;
   });
 };
+
+exports.insertComment = (id, properties) => {
+  const { username, body } = properties;
+  // console.log(username);
+  //need to go into table for users and check if username exists
+
+  return db
+    .query(`SELECT * FROM users WHERE username = $1`, [username])
+    .then((results) => {
+      if (results.rowCount > 0) {
+        return db
+          .query(
+            "INSERT INTO comments (body, author, review_id) VALUES ($1, $2, $3) RETURNING *",
+            [body, username, id]
+          )
+          .then((result) => {
+            console.log(result.rows[0]);
```

Comment:
lets remove this console log. Also we need to make sure we send back comment as an object not an array as it is singular when in a post.